### PR TITLE
[mod_feed]Code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_feed
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_feed/mod_feed.xml [hunk 1]
```diff
@@ -25,118 +25,124 @@
...
+
 				<field
 					name="rssitems"
 					type="text"
-					default="3"
 					label="MOD_FEED_FIELD_ITEMS_LABEL"
-					description="MOD_FEED_FIELD_ITEMS_DESC" />
+					description="MOD_FEED_FIELD_ITEMS_DESC"
+					default="3"
+					/>
```
Comment (andrepereiradasilva) [inline modules/mod_feed/mod_feed.xml]: 1 tab to high.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 5210e27f194db50add16e322aaa45735ee77e107<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10581">issues.joomla.org/joomla-cms/10581</a>.</sub>

**Comment by JoomliC** [comment]
I have tested this item :white_check_mark: successfully on 5210e27f194db50add16e322aaa45735ee77e107<br /><br />On code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10581">issues.joomla.org/joomla-cms/10581</a>.</sub>

**Comment by brianteeman** [comment]
Rtc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10581">issues.joomla.org/joomla-cms/10581</a>.</sub>

**Pull request closed without merging**